transmart-core-db/grails-app/domain/org/transmartproject/db/i2b2data/ObservationFact.groovy

```diff
@@ -69,6 +67,7 @@ class ObservationFact implements Serializable {
     static belongsTo = [
         patient      : PatientDimension,
         trialVisit   : TrialVisit,
+        // visit        : VisitDimension
```

Comment:
can this line be removed?

Reply:
It can, but this shows the desired state, which sadly is currently not possible, because encounter_num is part of the primary key, while the visit dimension is intended to be optional (nullable).